doc/design/diagrams/iceoryx_components_diagram_v3_0_0.puml

```diff
@@ -27,26 +28,42 @@ skinparam frame {
 
 header Eclipse iceoryx component overview
 
+' Platform Library containing platform abstraction with quality level 1
```

Comment:
Do we have them defined anywhere? Are those the ros quality level, if so I would name it ros quality level 1

Also for the other remarks in this PR

Reply:
@elfenpiff Yep, we have slightly adapted the ROS quality levels in the `CONTRIBUTING.md`.